src/index.html

```diff
@@ -15,8 +15,44 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo">
+    <img src="images/logo.png" alt="Moyo logo" class="logo"/>
+      </a>
```

Comment:
Remember about correct indentation between parent and child elements. Each level of nesting should have a 2-space offset.